test/3rd-cases/mock.js

```diff
@@ -1,3 +1,3 @@
-'use strict';
+'use strict'
 
-exports.test = 1;
```

Comment:
Nit (unrelated to this PR): I think we should finally get rid if files that are not part of any test suite and where it's not clear what they have ever been good for.
All files in this folder are an absolute riddle to me.